packages/core/src/components/card/_overrides.scss

```diff
@@ -0,0 +1,47 @@
+.#{$ns}-card {
+  border: 1px solid $card-background-color;
+
+  &.#{$ns}-dark,
+  .#{$ns}-dark & {
+    border: 1px solid $dark-card-background-color;
+  }
+
+  &:hover {
+    border-color: $card-hover-border-color;
+
+    &.#{$ns}-dark,
+    .#{$ns}-dark & {
+      border: 1px solid $dark-card-hover-border-color;
+    }
+  }
+
+  &.#{$ns}-disabled {
+    pointer-events: none;
+
+    > * {
+      opacity: .85;
+    }
+
+    &.#{$ns}-dark,
+    .#{$ns}-dark & {
+      > * {
+        opacity: .6;
+      }
+    }
+
+    &.#{$ns}-interactive:hover {
+      box-shadow: none;
+      cursor: auto;
+    }
+  }
+
+  &.#{$ns}-kind-g-active,
+  &.#{$ns}-kind-g-active:hover {
+    border-color: $g-primary3;
+  }
+
+  &.#{$ns}-kind-g-error,
+  &.#{$ns}-kind-g-error:hover {
+    border-color: $g-error-600;
```

Comment:
We are adding the custom classes in this [classes file](https://github.com/graphext/blueprint/blob/graphext/packages/core/src/common/classes.ts#L382). It helps to know what custom classes we have in each component and make it easier to use them in graphext.

Reply:
Right! Done with [87c8390](https://github.com/graphext/blueprint/pull/61/commits/87c83900f543de32e6510f4ff85edc2bbfee3dd6)